Applications/Forward/test/pendulum_no_gravity.osim

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <OpenSimDocument Version="20302">
-	<Model name="pendulum_no_gravity">
+	<Model name="pendulum">
 		<credits> Model authors names.. </credits>
```

Comment:
Any reason to change the name here? Is it diffed in the test case?

Reply:
The model name appears in the column labels and we are trying to compare two results one with gravity and one without with gravity replaced by an external load.